examples/containers/multi/helpers/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# Change 
+ARG couchbase_server_image=couchbase/server:enterprise-6.6.3
+
+# Build the exporter
+FROM golang:1.17.2 as go_build
+RUN git clone https://github.com/couchbase/couchbase-exporter.git /opt/couchbase-exporter
```

Comment:
It'd be good if you could override the version to use - for example, look at https://github.com/couchbaselabs/observability/blob/main/testing/resources/containers/cb-with-exporter.Dockerfile